src/Service/Player/Url.php

```diff
@@ -109,11 +109,11 @@ public function __toString()
      *
      * @see https://docs.theplatform.com/help/player-player-autoplay
      *
-     * @param bool $autoplay True to enable autoplay, false otherwise.
+     * @param bool $autoPlay True to enable autoPlay, false otherwise.
      */
-    public function setAutoplay(bool $autoplay)
+    public function setAutoplay(bool $autoPlay)
```

Comment:
If we are following camel case should this be setAutoPlay similar to the variable autoPlay?